apps/admin-gui/src/app/shared/components/user-detail-page/user-dashboard/user-dashboard.component.scss

```diff
@@ -0,0 +1,22 @@
+.dashboard-container {
+  columns: 300px 2;   //1. minimum width of column, 2. max number of columns
+  column-gap: 1rem;
+  margin: 0;
+  padding: 1rem;
```

Comment:
I would remove the padding. Without it, the content is nicely aligned with the title.